config/services.yaml

```diff
@@ -15,6 +15,8 @@ parameters:
     app.cache.ttl.get_user_with_assignments: '%env(int:CACHE_TTL_GET_USER_WITH_ASSIGNMENTS)%'
     app.cache.ttl.lti_instance_list: '%env(int:CACHE_TTL_LTI_INSTANCES)%'
     app.lti.load_balancing_strategy: '%env(resolve:LTI_INSTANCE_LOAD_BALANCING_STRATEGY)%'
+    app.lti.version: '%env(resolve:LTI_VERSION)%'
+    app.lti.registration_id: '%env(resolve:LTI1P3_REGISTRATION_ID)%'
```

Comment:
optional: what about `app.lti1p3.registration_id` since the concept of registrations applies to 1.3 ?

Reply:
Fixed